notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb

```diff
@@ -1,1369 +1,1349 @@
 {
```

Comment:
<p>Add new lines for each section.</p><p>1)..</p><p>2)...</p><p> </p><p>Also, add new line for each prediction type</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>